src/clients/api/queries/useGetVaults/useGetVestingVaults/index.ts

```diff
@@ -150,7 +150,9 @@ const useGetVestingVaults = ({
         (acc, _item, poolIndex) => {
           const totalStakedWeiData = poolBalances[poolIndex];
           const lockingPeriodMs = poolData[poolIndex]?.poolInfos.lockingPeriodMs;
-          const userStakedWei = poolData[poolIndex]?.userInfos?.stakedAmountWei;
+          const userStakedWei = poolData[poolIndex]?.userInfos?.stakedAmountWei.minus(
```

Comment:
Question: from a UX perspective, wouldn't this change potentially confuse users? It might be a bigger discussion, but I'd add the amount of pending withdrawals to the XVS (if greater than 0) vault card along with this change